### DISCUSSION TITLE
GTA V crossover on m1 max

### DISCUSSION THREAD

**Discussion by Lipooooo** [Heroic-Games-Launcher/HeroicGamesLauncher#1087] [Q&A]
hey guys,  i installed gta V with the help of crossover and heroic, everything works fine but my fps is around only 50, i saw a video of andrew where with an M1 max he managed to get 120 fps in normal 1080p quality how can i do to get more fps at least around 80
ty sorry for my english i m french

**Comment by elisezhu123** [comment]
it only work with the gta from steam. epic need more virtualization than steam, it will lose more fps
